src/standalone/UIKit/InputControls/CurrencyInput.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { TextFieldProps } from "@material-ui/core";
+import TextFieldWithHelp from "../TextFieldWithHelp";
+import { UIInputProps } from "../CommonStyles";
+import { getGlobalized } from "../../../globalize";
+
+export interface CurrencyInputProps extends UIInputProps {
+	/**
+	 * The text for info icon
+	 */
+	infoText?: React.ReactNode;
```

Comment:
why is `infoText` and `important` specified on these property type defs? just `extends TextFieldWithHelpProps`